workspace/python/ds-pack/2.1.0-base-python-3.10.13/Dockerfile

```diff
@@ -87,6 +87,26 @@ RUN curl -fsSL https://code-server.dev/install.sh \
     | sh -s -- --version 4.16.1 \
     && pip install --no-cache-dir ./codeserver-proxy
 
+# vscode cli
+RUN curl -Lk 'https://code.visualstudio.com/sha/download?build=stable&os=cli-alpine-x64' --output vscode_cli.tar.gz \
+    && tar -xf vscode_cli.tar.gz \
+    && cp ./code /usr/local/bin \
+    && rm -rf vscode_cli.tar.gz
+
+COPY vscode-tunnel vscode-tunnel
+
+RUN mkdir /vscode-tunnel/logs \
+    && mkdir /vscode-tunnel/docs \
+    && chmod -R 777 vscode-tunnel
+
+RUN cp vscode-tunnel/scripts/start.sh /usr/bin/start-tunnel \
+    && cp vscode-tunnel/scripts/stop.sh /usr/bin/stop-tunnel \
+    && cp vscode-tunnel/scripts/refresh.sh /usr/bin/restart-tunnel \
+    && cp vscode-tunnel/scripts/status.sh /usr/bin/tunnel-status \ 
+    && chmod +x /usr/bin/start-tunnel /usr/bin/stop-tunnel /usr/bin/restart-tunnel /usr/bin/tunnel-status
+
+COPY TUNNELING.md /vscode-tunnel/docs/README.md
```

Comment:
This isn't visible to the user. Unless we explicitly let them know, they won't know that this is present in the workspace.

Reply:
Not sure how we can make this possible though

Reply:
We will have to do something about this before we release the image otherwise user won't know how to manage the tunnel, there won't be any detail anywhere